lizmap/modules/admin/controllers/logs.classic.php

```diff
@@ -45,15 +45,36 @@ public function index()
         $conditions = jDao::createConditions();
         $detailNumber = $dao->countBy($conditions);
 
-        // Get last error log
-        $errorLog = \Lizmap\App\FileTools::tail(jApp::logPath('lizmap-admin.log'), 50);
+        // Number of lines for logs
+        $maxLinesToFetch = 200;
+
+        // Get last admin log
+        $lizmapLogPath = jApp::logPath('lizmap-admin.log');
+        $lizmapLog = \Lizmap\App\FileTools::tail($lizmapLogPath, $maxLinesToFetch);
+        $lizmapLogTextArea = $this->logLinesDisplayTextArea($lizmapLog);
+
+        $errorLogDisplay = \lizmap::getServices()->hideSensitiveProperties();
```

Comment:
wrong assignement to $errorLogDisplay. hideSensitiveProperties() indicate if we hide sensitive properties. So, if yes, we must **not** show errors log.

`$errorLogDisplay = ! \lizmap::getServices()->hideSensitiveProperties();`